content/articles/The-concept-of-cookies-in-servlets/index.md

```diff
@@ -0,0 +1,222 @@
+- `String getName()`: Returns the cookie's name, which cannot be modified after it has been created.
+- `String getValue()`: This method aids in obtaining the cookie's current value.
+- `int getVersion()`: Cookie's version number is returned by this method.
+- `void setVersion()`: Cookie's version number is set in this method.
+- ` int getMaxAge()`: This method is used to determine a cookie's maximum age.
+- `String getPath()`: The path from the cookie is obtained using this method.
+- `void setPath()`: This method is used to specify a cookie path. In a web application, for example, it is possible to specify a specific path for the cookies to be saved.
+- `String getDomain()`: This method can be used to extract the domain name from a cookie.
+ 
+### How to create, delete, and get cookies
+
+#### Creating a new Cookie object
+
+```java
+Cookie object=new Cookie("uName",variableName);
+```
+#### Setting up lifespan for a cookie
+
+```java
+object.serMaxAge(20*60); // setting maximum age of the cookie.
+
+```
+#### Sending the cookies to the client
+
+```java
+response.addCookie(object); // adding cookie to the response object.
+```
+
+#### Getting cookie from a client request
+
+```java 
+Cookie[] object=request.getCookies();
+```
+#### Deletting cookies
+
+```java
+Cookie object=new Cookie("userId",""); 
+ck.setMaxAge(0);
+response.addCookie(object);
+
+```
+
+### Example to demonstrate Servlet Cookie
+We'll develop two servlet classes in this example, one to create cookies and the other to retrieve them. Here we will create two cookies to hold the user name and the allocated user subject, respectively.
+
+In this scenario, the JSP page gives us a form to fill out with our information, such as user name and subject.
+
+#### Steps to create this Cookies project:
+
+#### Step 1: Creating the Example to demonstrate cookies in servlet project.
+Open Eclipse or any other Java IDE, create a new Dynamic web project by selecting a new Dynamic web project. Provide your favorite project name. In my case, I will call it `Example to demonstrate cookies in servlet`.
+#### Step 2: Creating index.JSP page 
+By right-clicking on `Example to demonstrate cookies in servlet` and selecting New JSP, you can code a JSP page as shown. Provide your JSP class name. I will call it `index.jsp`.
+
+```JSP
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="ISO-8859-1">
+    <title>Insert title here</title>
+</head>
+
+<body>
+    <form action="ProcessOneServlet">
+        Enter User name <input type="text" name="userName" required><br>
+
+        Enter subject <input type="text" name="subject" required><br>
+        <input type="submit" value="Click me">
+    </form>
+</body>
+
+</html>
+```
+
+#### Step 3: Creating a ProcessOneServlet Servlet page
+By right-clicking on the`Example to demonstrate cookies in servlet` project selecting New Servlet, you can code a Servlet page as shown below. Provide your Servlet class name. I will call it `ProcessOneServlet`.

```

Comment:
Change to "You can code a Servlet page as shown below by right-clicking on the`Example to demonstrate cookies in servlet` project selecting New Servlet. First, provide your Servlet class name. I will call it `ProcessOneServlet`."